halyard-config/src/main/java/com/netflix/spinnaker/halyard/config/model/v1/providers/openstack/OpenstackProvider.java

```diff
@@ -16,4 +19,19 @@ public ProviderType providerType() {
   public void accept(ConfigProblemSetBuilder psBuilder, Validator v) {
     v.validate(psBuilder, this);
   }
+
+  @Override
+  public OpenstackBakeryDefaults emptyBakeryDefaults() {
+    OpenstackBakeryDefaults result = new OpenstackBakeryDefaults();
+    // Set defaults over here
+    result.setAuthUrl(null);
+    result.setFloatingIpPool(null);
+    result.setInsecure(false);
+    result.setNetworkId(null);
+    result.setPassword(null);
```

Comment:
Setting `null` isn't needed, you can just return `new OpenstackBakeryDefaults()`